### ISSUE TITLE
[Feature Request] Allow pasting files directly into the page

### ISSUE THREAD

**Issue by JeremyKennedy** [szabodanika/microbin#94]
I would like to be able to take a screenshot (which is copied to my clipboard), and paste it directly into microbin. That way I don't need to save it to my PC and select the file.

**Comment by luxus** [comment]
@szabodanika are you already working on it? maybe we create a PR for it

**Comment by szabodanika** [comment]
No, not just yet. I am also not quite sure how we would go about this. Can this be done with simple vanilla JS? If so then I am very happy with you working on a PR for this

**Comment by JeremyKennedy** [comment]
Good news - It seems fairly straightforward with vanilla JS!

https://stackoverflow.com/questions/6333814/how-does-the-paste-image-from-clipboard-functionality-work-in-gmail-and-google-c

https://stackoverflow.com/questions/490908/paste-an-image-from-clipboard-using-javascript

https://codepen.io/ahw/pen/qEPRzW

I have a ton of experience with JS, though none with Rust, so not sure how helpful I'd be on this one.

**Comment by fireynis** [comment]
@szabodanika I also had the same feature thought, and being a dev I figured I could knock it out quick, and I did! Take a look here if interested: https://github.com/szabodanika/microbin/pull/332
